### DISCUSSION TITLE
Disable Efficiency Mode

### DISCUSSION THREAD

**Discussion by BobbyShoe** [MicrosoftEdge/WebView2Feedback#4021] [Q&A]
Is there anyway to disable efficiency mode in WebView2?  The menu options in Windows task manager related to efficiency mode appear to be disabled for WebView2 processes.

**Comment by novac42** [comment]
Thanks for reaching out. I've assigned this to a dev that can best answer your question.

**Comment by aluhrs13** [comment]
@BobbyShoe - Efficiency mode is managed by WV2 for some of its subprocesses depending on the visibility of your WV2 control(s). Many of the shared processes will never go into efficiency mode for a variety of reasons, but renderer processes (generally one per WV2 instance) will go into efficiency mode.

If you want to avoid efficiency mode, keep your WV2 control `IsVisible` property set to `true`.

**Comment by LummoxJR** [comment]
This still needs resolution. Efficiency mode needs a way to be disabled application-wide, because users with certain processors have been experiencing application stuttering.

Also I need to add that the information about an IsVisible property is obviously only applicable to .NET, but doesn't tell anyone in C++ land how to solve the problem. I did find where I can call put_isVisible() in COM, but here we have an additional problem: My application is never setting that to false, so the browser should always be visible already. Therefore it also appears that the information about keeping browsers visible to prevent efficiency mode is incorrect.

Can a browser flag or setting be added for this?
